sql.txt

```diff
@@ -1,15 +1,23 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
+SELECT CustomerName, ContactName, Country FROM Customers ORDER BY Country ASC LIMIT 11 OFFSET 20;
```

Comment:
This exercise is requiring to get entries from 11 to 20 (11, 12, 13, ... 19, 20). With LIMIT 11 OFFSET 20, we are starting at position 21 and we are getting 11 rows (21, 22, 23, ..., 30, 31).